client/src/store/index.ts

```diff
@@ -1,23 +1,26 @@
 import Vue from 'vue';
 import Vuex from 'vuex';
 
-import * as simulation from './modules/simulation';
 import * as filtres from './modules/filtres';
-import { models } from './modules/models';
-import { query } from './modules/query';
+import * as knowledgeGraph from './modules/knowledgeGraph';
+import * as models from './modules/models';
+import * as query from './modules/query';
+import * as simulation from './modules/simulation';
 
 Vue.use(Vuex);
 
 const store = new Vuex.Store({
   modules: {
-    simulation,
     filtres,
+    knowledgeGraph,
```

Comment:
So we have a `models` store and a `knowledgeGraph` store. Maybe we can actually change the name of the `models` store to singular --> `model` to be consistent.